### PR TITLE
Ubuntu Touch: Updates

### PR DESCRIPTION
**Pull request by lduboeuf**

### CODE REVIEW

File: src/core/GCAudio.qml [hunk 1]
```diff
@@ -214,7 +214,10 @@ Item {
             source = ""
             gcaudio.done()
         } else {
-            source = ""
+            // on Ubuntu Touch, emptying the source result in an Audio error which triggers that method again endlessly
```
Comment (petitlapin) [inline src/core/GCAudio.qml:217]: should it be the same on line 140?

Comment (lduboeuf) [reply src/core/GCAudio.qml:217]: idk, but haven't spot any audio issue with it. Need to dig a bit to know the difference between those two

**Comment by petitlapin** [comment]
Hi,

I've pushed both commit in https://invent.kde.org/education/gcompris/-/commit/d8448c704348555d804b236d0c9cdcec0f470a57 and https://invent.kde.org/education/gcompris/-/commit/e06a55df3e732855e23a888f21d1130e7fa1e99f

File: src/core/GCAudio.qml (hunk 1 above)
Comment (petitlapin) [reply src/core/GCAudio.qml:217]: If I remember correctly, "play" is called to play the sound directly, while the other one is when you append audio to play after the current one has finished.

Intro audio and when you lose on an activity (for example reversecount) will play using "play". You can lose twice and it should be the same audio (try_again) playing twice in a row

**Pull request closed without merging**